Bio/PDB/PDBList.py

```diff
@@ -271,16 +269,16 @@ def retrieve_pdb_file(
 
         # Get the compressed PDB structure
         pdb_code = pdb_code.lower()
-        archive = {
+        archive_dict = {
             "pdb": f"pdb{pdb_code}.ent.gz",
             "mmCif": f"{pdb_code}.cif.gz",
             "xml": f"{pdb_code}.xml.gz",
             "mmtf": f"{pdb_code}",
             "bundle": f"{pdb_code}-pdb-bundle.tar.gz",
         }
-        archive_fn = archive[file_format]
+        archive = archive_dict[file_format]
 
-        if file_format not in archive.keys():
+        if file_format not in archive_dict.keys():
             raise Exception(
                 f"Specified file_format {file_format} doesn't exists or is not supported. Maybe a "
                 "typo. Please, use one of the following: mmCif, pdb, xml, mmtf, bundle"
```

Comment:
You can also print the keys of `archive_dict` here instead of listing the formats manually. One less place to update if the dict changes.